tests/test_models.py

```diff
@@ -87,18 +87,36 @@ def test_last_theme_activated_on_multiple_themes_activated(self):
                 self.assertEqual(Theme.get_active_theme().pk, theme.pk)
         self.__test_active_theme()
 
-    def test_use_db_defined_in_kwargs(self):
-        Theme.objects.all().delete()
-        theme_1 = Theme.objects.create(name="Custom 1", active=True)
-        kwargs = {"using": "default"}
-        Theme.get_active_theme(**kwargs)
+    def test_repr(self):
+        theme = Theme.get_active_theme()
+        self.assertEqual("{0}".format(theme), "Django")
```

Comment:
This is `test_repr`, so the test should check `repr(theme)`.

String formatting calls `str`, which uses `__str__` with a fallback to `__repr__`, but that’s not obvious and tests should be obvious 🙂